docs/imagecustomizer/api/configuration/input.md

```diff
@@ -4,18 +4,36 @@ parent: Configuration
 
 # input type
 
-Specifies the configuration for the input image.
+Specifies the configuration for the input image and associated files required for customizing.
 
 Example:
 
 ```yaml
 input:
   image:
     path: ./base.vhdx
+  verityHashes:
+    path: /input/verityhashes
```

Comment:
how do the contents of this path map to individual partitions?

Reply:
The path is for the host, where Prism should look for the signed files. Then the input path is kind of default, like we always want to input UKI to esp partition.